templates/config.yml.j2

```diff
@@ -1,3 +1,5 @@
 url: {{ theo_url }}
 token: {{ theo_client_token }}
 cachedir: {{ theo_agent_cache_dir }}
+verify: {{ theo_verify_signature|capitalize }}
+{{ theo_public_key_path }}
```

Comment:
Should't this be `theo_config_public_key_path`? or whatever we choose to call this variables (see above)

Reply:
good catch